### DISCUSSION TITLE
【委托】给自家邦布找相亲对象

### DISCUSSION THREAD

**Discussion by aethna** [share121/inter-knot#2427] [其他]
邦布长相如图所示：
![2a61cb50b2a0deaf4bbb57ec04884cef](https://github.com/user-attachments/assets/536e6c76-7703-4888-a12d-1a8e662c4aa9)

该邦布名字叫鲨呜，已经陪伴我十年了，顺带一提他，喜欢充海鱼口味的电。自从他看了那个影片卡尔一家后就疯了似的想组建一个家庭，原本是想拒绝的，但是他毕竟还是陪了我十年，也是该为他找个伴了。所以如果有愿意的网友可以与我私聊。或者来十二分街与我线下见面。
ps：如果邦布主人是漂亮小姐姐就更好了

**Comment by YuTinMIn** [comment]
给邦布找怎么还要主人是漂亮小姐姐呢，你小子，肯定有私情……

**Comment by Genshin-Neuvillette** [comment]
直接月吧，我给你资源

**Comment by WORLD-executeME** [comment]
你到底是想给邦布还是自己找个伴啊
